obs-package-status/README.md

```diff
@@ -61,5 +61,10 @@ A list of OBS Repositories you want to check the package for. Separated by blank
 
 ## Usage
 1. Clone the repository
-1. bundle install
-1. bundle exec status-check.rb
+2. Build the docker container:
+  ```docker build -ti obs-package-bot .```
+3. Copy the obs-package-bot.service to /etc/systemd/system
+4. Start the service:
+```
+docker run --restart always -v $PWD:/home/package-bot/obs-package-status -e RUN_EVERY=600 -d obs-package-status
```

Comment:
If there is one or more spaces in the current path, this command will fail. To prevent this, surround `$PWD` with double quotes. The command would then be:

```
docker run --restart always -v "$PWD:/home/package-bot/obs-package-status" -e RUN_EVERY=600 -d obs-package-status
```

Reply:
done